modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -131,6 +131,14 @@ enum class Feature(
         hasFirebaseRemoteFlag = false,
         hasDevToggle = true,
     ),
+    GRAVATAR_CHANGE_AVATAR(
```

Comment:
iOS folks [have requested](https://github.com/Automattic/pocket-casts-ios/pull/1770#pullrequestreview-2186049325) for an FF to control the `Change Avatar` button. We are adding the same FF on Android.

As this is my first FF on the PocketCast repo, please let me know if there is something wrong with the configuration. We want to be able to disable the button remotely, so I would say `hasFirebaseRemoteFlag` should be `true`.

Reply:
Configuration is fine but we'll need to add this flag to the Firebase Remote Config as well.

Reply:
Do we have some docs or a flow to request this?

Reply:
No, not really. It's just adding an entry to Firebase. I've sent you and invitation. DM me if anythings gets confusing.

Reply:
Done! Thanks for your help @MiSikora !